### DISCUSSION TITLE
[Launchpad] Can't replace XMR or Wallet Address

### DISCUSSION THREAD

**Discussion by brianp** [tari-project/tari#6230] [LaunchPad] [answered]
Source: 
- [Telegram](https://t.me/tariproject/72733) _i can't replace xmr address_
- [Discord](https://discord.com/channels/892738558316662855/894350744419962890/1222757102267666443) _...and the app does not let you change the Monero address, which looks to have some other setup text that looks like it came from my clipboard_
- [Telegram](https://t.me/c/2101648014/2863) _Can we change the monero mining address in the app or is it default till Tari launches_

**Comment by hansieodendaal** [comment, marked as answer]
In Launchpad, you can only change editable fields if no Docker processes are running:
- change a filed just after the application started;
- stop all running processes, then change the values.
